src/EasyList/Validate.cs

```diff
@@ -0,0 +1,76 @@
+﻿using EasyList;
+using EasyList.DataModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class Validate
+{
```

Comment:
I wouldn't make this static. You should create a new instance of the validator each time you want to run it.

Reply:
I not able to understand the reasoning.

Reply:
The Validate class validates one object. It does so by having a field called errors. Now the errors field is not for every validated item but rather for the one being validated right now. So you use one Validator to validate an item, then you use another validator for another. Because every validator that runs has different errors due to another object meaning there should never be just one list of errors.